Assets/Scripts/Blobs/BlobMoveDirection.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
```

Comment:
Is this class necessary? If not, please delete. If it is, please fix:
* Remove unnecessary `using` statements
* There should be no new line before opening curly braces
* Empty methods can be removed or condensed to one line, e.g.`Start() { }`